### DISCUSSION TITLE
Where are donation links?

### DISCUSSION THREAD

**Discussion by Fox2Code** [CuarzoSoftware/Louvre#25] [General]
While going though this project, I didn't found any donation links to support the project.

Just want to know what is planned for donations, or if a donation link already exists if I missed it.

**Comment by ehopperdietzel** [comment]
Hello, thank you for asking. Some time ago, I had activated GitHub sponsors, which I believe accepts donations, but I deactivated it when Louvre started to gain more attention because I didn't want to give the impression that I do this for money. But now that I want more people to contribute to its development, distributing those donations among the contributors based on their contributions could be an incentive. Therefore, I will see if I can reactivate it.

**Reply by Fox2Code** [reply]
@ehopperdietzel I see GitHub sponsors are still not enabled, are you having an issue enabling it?

Do you have a Ko-fi or PayPal? Because for now it's very hard to give you money!

If you don't feel like doing something fancy, that fine, you can just send your username and the website you want money on.

**Reply by ehopperdietzel** [reply]
Hi again! I believe I've enabled it for @CuarzoSoftware. GitHub indicates that the sponsors page is publicly accessible [here](https://github.com/sponsors/CuarzoSoftware). Can you confirm if it's visible to you? Additionally, I do have a PayPal account, but I haven't used it in years, so I might need to search for the credentials haha.

**Reply by Fox2Code** [reply]
Seems like GitHub sponsors have too many necessary frictions, which may limit the number of peoples financially contributing to CuarzoSoftware/Louvre.

Any amount of friction can reduce the number of financial supporters, I dealt with it cause Louvre look really nice, but not everyone will do the same.

**Reply by ehopperdietzel** [reply]
I noticed that you are a sponsor of @CuarzoSoftware. Thank you very much ❤️, I truly appreciate it! I'll explore how to display that in the repository. As for what you mentioned, I wasn't aware of the difficulty of donating via GitHub. Which platform would you recommend? Ko-fi and PayPal?

**Reply by Fox2Code** [reply]
Ko-Fi and PayPal have low fees, having multiples way of donating would probably increase donations.
GitHub doesn't allow peoples to sponsor using PayPal, so yeah, I recommend doing both PayPal and Ko-Fi.
I usually sponsor via Patreon, but since I know the fees are a bit higher there, it's for you to decide if it's worth it.

**Reply by ehopperdietzel** [reply]
Okay, I'll keep that in mind, and thanks again!
